### DISCUSSION TITLE
Possible to Torrent the z_xxx_yy files?

### DISCUSSION THREAD

**Discussion by BinaryOutlook** [kubilus1/autoortho#150] [Ideas]
The GitHub download speed currently is relatively low(few mb/s) if we torrent it, it would likely speed up the installation process. 

* Just a consideration, maybe the owner's local law doesn't allow torrent.

**Comment by kubilus1** [comment]
Yeah this isn't a bad idea.  Could save load on Github servers as well.  Not sure quite how it'd be integrated though.

**Reply by BinaryOutlook** [reply]
How does this sound? GitHub download will still be the default, however, one may opt for Torrent download and drag the files into custom scenery

**Comment by jonaseberle** [comment]
Without any changes to autoortho needed, you can already download them and put them in the /downloads folder and they will be used for installation.

**Comment by gningue25** [comment]
Installation use files from DOWNLOAD folder
